pkg/common/utils_test.go

```diff
@@ -706,70 +706,82 @@ func TestConvertMiBStringToInt64(t *testing.T) {
 		expectError bool
 	}{
 		{
-			"valid number string",
-			"10000",
-			1,
-			false,
+			desc:        "valid number string",
+			inputStr:    "10000",
```

Comment:
why is 10000 converted to 1 and 1000Ki is also 1?

Reply:
The round up is doing roundedUp := volumeSizeBytes / allocationUnitBytes(Here we use MiB which is int 1048576)

10000 / 1048576 round up is 1
1000Ki / 1048576 round up is 1
1000k / 1048576 round up is 1 as well.